LibGit2Sharp/Repository.cs

```diff
@@ -656,7 +673,24 @@ internal Commit LookupCommit(string committish)
         /// <returns>The references in the remote repository.</returns>
         public static IEnumerable<Reference> ListRemoteReferences(string url)
         {
-            return ListRemoteReferences(url, null);
+            string _defaultBranch;
+            return ListRemoteReferences(url, null, out _defaultBranch);
```

Comment:
You can get rid of the  variable and just pass a discard:

return ListRemoteReferences(url, null, out _);

Reply:
Yeah I know, but yet again, this codebase seems to be old enough to require the older constructs, so I decided to leave it be like this. But again, please let me know if you feel we're at the point when we can start deviating from common patterns in this code. The reason why I try to follow them is because I'm still not sure if we won't be contributing back.